src/main/java/com/example/application/models/ContractsEntity.java

```diff
@@ -0,0 +1,67 @@
+package com.example.application.models;
+
+import javax.persistence.*;
+import java.util.Objects;
+
+@Entity
+@Table(name = "contracts", schema = "public", catalog = "postgres")
+public class ContractsEntity {
+    @GeneratedValue(strategy = GenerationType.IDENTITY)
+    @Id
+    @Column(name = "id")
+    private int id;
+    @Basic
+    @Column(name = "phoneNumber")
+    private String phoneNumber;
+    @Basic
+    @Column(name = "planID")
+    private int planId;
+    @Basic
+    @Column(name = "clientID")
+    private int clientId;
```

Comment:
You should join this column with ID from ClientsEntity. Also, you should do this for other Entities due to your db design.
Example:
https://www.baeldung.com/hibernate-one-to-many

PS: maybe you need some other relationships